src/main/scala/files_metadata/infrastructure/MetadataRoutes.scala

```diff
@@ -146,6 +146,19 @@ case class MetadataRoutes() extends cask.Routes {
       controllers.UnShareFileController( request, ownerUUID, fileUUID )
     )
   }
+  private val deleteMetadataEndpoint =
+    s"$basePath/delete/:ownerUUID/:fileUUID"
```

Comment:
According to the [spec](https://github.com/hawks-atlanta/metadata-scala/blob/dev/docs/spec.openapi.yaml), the endpoint should be `$basePath/:ownerUUID/:fileUUID` (without the delete).

Reply:
Repaired:

Test prefix
![image](https://github.com/hawks-atlanta/metadata-scala/assets/47455237/637f8fab-f6b9-4352-a33b-7cee60c4c044)


MetadataRoutes
![image](https://github.com/hawks-atlanta/metadata-scala/assets/47455237/54ecc875-8cca-4f6d-8e72-0892dd34507a)

Reply:
Check why the tests are failing

Reply:
I can't think of anything about this. Is successful with s"$basePath/delete/:ownerUUID/:fileUUID" but not with s"$basePath/:ownerUUID/:fileUUID".
what do you think?

Reply:
Keep the `/delete` in the endpoint and update the spec instead.

Reply:
Rpaired:
![image](https://github.com/hawks-atlanta/metadata-scala/assets/47455237/71d5961b-fc92-4453-8f33-cef715ccf8f4)